File: SNS.Web/src/app/jobs/company-create-requests/contracts/company-create-request-summary.dto.ts
import { CompanyCreateRequestStatus } from '../../enums/company-create-request-status.enum';

export interface CompanyCreateRequestSummaryDto {
    id: string;
    profileId: string;
    profileFullName: string;
    name: string;
    industry: string;
    status: CompanyCreateRequestStatus;
    createdAt: string;
    reviewedAt: string | null;
}

File: backend/SNS.Application/Jobs/CompanyCreateRequests/Contracts/CompanyCreateRequestSummaryDto.cs
using SNS.Domain.Jobs.Enums;

namespace SNS.Application.Jobs.CompanyCreateRequests.Contracts;

public sealed record CompanyCreateRequestSummaryDto(
    Guid Id,
    Guid ProfileId,
    string SubmitterName,
    string? SubmitterAvatarUrl,
    string Name,
    string Industry,
    string? WebsiteUrl,
    string? LogoUrl,
    CompanyCreateRequestStatus Status,
    DateTime CreatedAt,
    DateTime? ReviewedAt
);
